### PR TITLE
Remove `long` for python3 compat

### PR DESCRIPTION
**Pull request by ekaitz-zarraga**
Proposed a change for python3 compatibility improvement. #17 
Idea was to make as few changes as possible.
Added the new module in `others` folder because I thought it was the most suitable for this (might be better to move it to a different one?)

### CODE REVIEW

File: others/py3compat.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Workaround for Python3 compat
+import sys
+if sys.version_info > (3,):
```
Comment (cclauss) [inline others/py3compat.py]: https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection
```python
try:
    long        # Python 2
except NameError:
    long = int  # Python 3
```

File: exporters/SimpleEval.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
 import string
 
 from decimal import Decimal
+from others.py3compat import *
```
Comment (cclauss) [inline exporters/SimpleEval.py]: Can we avoid * imports?

Comment (ekaitz-zarraga) [reply exporters/SimpleEval.py]: Yes:
from others.py3compat import long, integer_types

File: others/py3compat.py (hunk 1 above)
Comment (ekaitz-zarraga) [reply others/py3compat.py]: good one, i'll chage that

**Comment by joxeankoret** [comment]
Already applied.

**Pull request closed without merging**